### PR TITLE
Merge from Develop to Main

### PR DESCRIPTION
**Pull request by carlosalcala-freestar**
## Description, Motivation and Context

**What is the goal of ticket? 
 
Merge from Develop to Main/Updated React plugin that has support for Thumbnail

## How Has This Been Tested? 

Manually tested App Build

## Does this change require updates outside of the code, like updating the documentation, adding a feature flag, or adding fields to a DB table? If so, was this done?

N/A

## Any tickets that this is dependent upon?

N/A

## Should reviewers manually test your changes?

N/A

### CODE REVIEW

**Comment by carlosalcala-freestar** [comment]
@maddapper @kkawai  working on fixing the conflicts

**Review by kkawai — APPROVED** [review]

File: FreestarAds-Ogury-PreRelease.template.local.podspec [hunk 1]
```diff
@@ -10,6 +10,8 @@ Pod::Spec.new do |spec|
   spec.vendored_frameworks  = 'build/FreestarAds-Ogury.xcframework'
   spec.dependency "FreestarAds#{prerelease_tag}", "~> 5.12#{beta_tag}"
```
Comment (maddapper) [inline FreestarAds-Ogury-PreRelease.template.local.podspec]: @carlosalcala-freestar, is this correct?  is core version 5.12 compatible with ogury adapter?

File: FreestarAds-Ogury.template.podspec [hunk 2]
```diff
@@ -10,6 +10,8 @@ Pod::Spec.new do |spec|
   spec.vendored_frameworks  = 'build/FreestarAds-Ogury.xcframework'
   spec.dependency "FreestarAds#{prerelease_tag}", "~> 5.12#{beta_tag}"
```
Comment (maddapper) [inline FreestarAds-Ogury.template.podspec]: @carlosalcala-freestar, is this correct?  is core version 5.12 compatible with ogury adapter?

**Review by maddapper — COMMENTED** [review]
@carlosalcala-freestar see comments

File: FreestarAds-Ogury.template.podspec (hunk 2 above)
Comment (carlosalcala-freestar) [reply FreestarAds-Ogury.template.podspec]: Fixed

File: FreestarAds-Ogury-PreRelease.template.local.podspec (hunk 1 above)
Comment (carlosalcala-freestar) [reply FreestarAds-Ogury-PreRelease.template.local.podspec]: Fixed

**Comment by carlosalcala-freestar** [comment]
@kkawai @maddapper  review again please

**Review by maddapper — APPROVED** [review]

**Pull request merged**